client/src/components/common/AlertSnackbar.js

```diff
@@ -0,0 +1,29 @@
+import { Alert, Snackbar } from '@mui/material'
+import PropTypes from 'prop-types'
+
+AlertSnackbar.propTypes = {
+  open: PropTypes.bool.isRequired,
+  handleClose: PropTypes.func.isRequired,
+  message: PropTypes.string.isRequired,
+  severity: PropTypes.string,
+}
+
+export function AlertSnackbar({
+  open,
+  handleClose,
+  message,
+  severity = 'error',
+}) {
+  return (
+    <Snackbar
+      anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
```

Comment:
```suggestion
      anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
```

Reply:
Well actually idk... it's a lot less noticeable at the bottom so maybe top is better?

Reply:
i prefer top